lib/model/frames/form.js

```diff
@@ -144,11 +144,12 @@ Form.Extended = class extends Frame.define(
   'excelContentType',   readable,
   // counts of submissions in various review states
   'receivedCount',                      'hasIssuesCount',
-  'editedCount'
+  'editedCount',                        'hasDataset',  readable
 ) {
   forApi() {
     return {
       submissions: this.submissions || 0,
+      hasDataset: this.hasDataset || false,
```

Comment:
You will need to add `hasDataset` to the form extended metadata API docs, though. Doesn't have to be right now, but we might want to make a note so we don't forget.

Reply:
will update API docs in next PR